src/components/Storage/Card/Table/Header/ActionHeader/ActionHeader.tsx

```diff
@@ -75,9 +76,11 @@ export const ActionHeader: React.FC<
               Open all files
             </Button>
             <Button
-              onClick={() => {
-                selection.clearAll();
-                return deleteFiles(paths);
+              onClick={async () => {
+                if (await confirmDeleteSelectedFiles()) {
+                  selection.clearAll();
+                  return deleteFiles(paths);
```

Comment:
We'll need to update the test here:
https://github.com/firebase/firebase-tools-ui/blob/bfff08543abf3fd62b36eda2a1f694567699859f/src/components/Storage/Card/Table/Header/ActionHeader/ActionHeader.test.tsx#L23